src/vehicle.cpp

```diff
@@ -4273,7 +4275,7 @@ veh_collision vehicle::part_collision( int part, const tripoint &p,
             }
         }
 
-        coll_velocity = vel1_a * ( smashed ? 100 : 90 );
+        coll_velocity = vel1_a * ( smashed ? 100 : 99 );
```

Comment:
Actually, see #19974 discussion - it was added for a reason. "Improved" is loaded.